Documentation/getting-started-ubuntu-trusty.md

```diff
@@ -16,11 +16,51 @@ vagrant up --provider virtualbox
 vagrant ssh
 sudo su
 
-curl -L https://github.com/coreos/rocket/releases/download/v0.1.1/rocket-v0.1.1.tar.gz -o rocket-v0.1.1.tar.gz
-tar xzvf rocket-v0.1.1.tar.gz
-cd rocket-v0.1.1
-./rkt fetch https://github.com/coreos/etcd/releases/download/v0.5.0-alpha.4/etcd-v0.5.0-alpha.4-linux-amd64.aci
+wget https://github.com/coreos/rocket/releases/download/v0.2.0/rocket-v0.2.0.tar.gz
+tar xzvf rocket-v0.2.0.tar.gz
+cd rocket-v0.2.0
+./rkt help
+```
+## Trust the CoreOS signing key
+
+This shows how to trust the CoreOS signing key with the procedure outlined in [Signing and Verification Guide](https://github.com/coreos/rocket/blob/master/Documentation/signing-and-verification-guide.md)
+
+Download the public key
+`curl -O https://coreos.com/dist/pubkeys/aci-pubkeys.gpg`
+
+Get the fingerprint
+```
+gpg --with-fingerprint aci-pubkeys.gpg
+```
+
+```
+pub  4096R/88182190 2015-01-23 CoreOS ACI Builder <release@coreos.com>
+Key fingerprint = 8B86 DE38 890D DB72 9186  7B02 5210 BD88 8818 2190
+```
 
+Format the fingerprint for the filename
+```
+echo "8B86 DE38 890D DB72 9186  7B02 5210 BD88 8818 2190" | tr -d "[:space:]" | tr '[:upper:]' '[:lower:]'`
+```
+
+```
+8b86de38890ddb7291867b025210bd8888182190
+```
+
+Create the directory to trust all images signed by this key
+```
+mkdir -p /etc/rkt/trustedkeys/root.d//etc/rkt/trustedkeys/root.d/8b86de38890ddb7291867b025210bd8888182190
```

Comment:
Could you change this to prefix rather than the root key? We don't want to tell people to trust the CoreOS key for everything :-)
`/prefix.d/example.com/`